code/game/machinery/scan_gate.dm

```diff
@@ -166,6 +166,7 @@
 		set_scanline("scanning", 10)
 
 /obj/machinery/scanner_gate/proc/alarm_beep()
+	say("Buzz!")
 	if(next_beep <= world.time)
```

Comment:
???

Reply:
I have a feeling that's on their main since it's in every one of their PRs.

#52451